bindings/generator/api/common.py

```diff
@@ -243,6 +243,21 @@ class SequesterVerifyKeyDer(BytesBasedType):
     pass
 
 
+class SecretKey(BytesBasedType):
+    pass
+
+
+class Bytes(BytesBasedType):
+    """
+    libparsec's bytes
+    """
+
+    custom_from_rs_bytes = (
+        "|b: &[u8]| -> Result<libparsec::Bytes,  String> { Ok(libparsec::Bytes::from(b.to_vec())) }"
```

Comment:
You may be able to use [Bytes::copy_from_slice(b)](https://docs.rs/bytes/latest/bytes/struct.Bytes.html#method.copy_from_slice) instead